src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/osd/osd-form/osd-form.component.ts

```diff
@@ -71,14 +79,25 @@ export class OsdFormComponent extends CdForm implements OnInit {
 
   hasOrchestrator = true;
 
+  simpleDeployment = true;
+
+  deploymentOptions: DeploymentOptions | undefined;
+  deploymentOptionNames: { [key in DeploymentOptionNames]: string };
```

Comment:
Almost the same as defined below? Duplicate code? Let's try to simplify a have everything defined clearly and once.

Reply:
Done!